codechecker_lib/arg_handler.py

```diff
@@ -157,12 +159,19 @@ def handle_server(args):
                     'checker_md_docs': checker_md_docs,
                     'checker_md_docs_map': checker_md_docs_map}
 
-    client_db_access_server.start_server(package_data,
-                                         args.view_port,
-                                         db_connection_string,
-                                         suppress_handler,
-                                         args.not_host_only,
-                                         context.db_version_info)
+    try:
+        client_db_access_server.start_server(package_data,
+                                             args.view_port,
+                                             db_connection_string,
+                                             suppress_handler,
+                                             args.not_host_only,
+                                             context.db_version_info)
+    except socket.error as err:
+        if err.errno == errno.EADDRINUSE:
+            LOG.error("Server can't be started, maybe the given port number "
```

Comment:
Also don't we usually stop the execution of codechecker after an error? Shouldn't there be a sys.exit here?